src/components/MapExplorer.jsx

```diff
@@ -1,38 +1,47 @@
 import React from 'react';
-import { Script } from 'react-loadscript';
-
-import Cameras from '../data/Cameras.jsx';
-
+import {Script} from 'react-loadscript';
 
 export default React.createClass({
+  //All CartoDB configuration data
+  cartodbConfig: {
+    mapVisualisationUrl: 'https://shaunanoordin-zooniverse.cartodb.com/api/v2/viz/e04c2e20-a8a9-11e5-8d6b-0e674067d321/viz.json',
+    dataTable: 'wildcam_gorongosa_cameras_201601',
+    dataLayerIndex: 1
+  },
 
-  //The Google API key ending with dld08s ('dildos') is tied to the
-  //shaun.a.noordin@zooniverse.org account and is only used for development.
-  GOOGLE_MAPS_API_KEY: 'AIzaSyAzMva99KALzUQfe_BCkCovZyGK-dld08s',
-  GOOGLE_MAPS_API_VERSION: '3',
-
-  googleMap: undefined,
+  cartodbVis: undefined,
+  cartodbMap: undefined,
+  cartodbLayers: undefined,  //Array of map layers. layer[0] is the base (cartographic map).
+  cartodbDataLayer: undefined,
 
-  getInitialState: function() {
-    this.googleMap = undefined;
+  getInitialState() {
+    this.cartodbVis = undefined;
+    this.cartodbMap = undefined;
+    this.cartodbLayers = undefined;
+    this.cartodbDataLayer = undefined;
     return {};
```

Comment:
If these properties need to be defined as the initial state, then they should be inside the return object, i.e.:

```
getInitialState() {
  return {
    cartodbVis: undefined,
    cartodbMap: undefined,
    cartodbLayers: undefined,
    cartodbDataLayer: undefined,
  }
}
```

...and then they'd be referred to under `this.state.foo`. This saves having to have a cleanup method as well, as the state is trashed when the component is unmounted.